### ISSUE TITLE
Dockerfile.dev and Dockerfile.prod

### ISSUE THREAD

**Issue by qyf13505049895** [egemenkopuz/temporal-bevfusion#11]
Thank you very much for your work, can you explain the difference between Dockerfile.dev and Dockerfile.prod when testing OSDaR23 data sets and which one should be used

**Comment by egemenkopuz** [comment]
I divided it into two parts: utilizing the .dev version exclusively for development and testing by passing my local project working directory through the volume parameter in Docker, avoiding the need to transfer all contents into the Docker container. Meanwhile, with the .prod version, everything is copied into Docker and the setup is executed via make command. So I would suggest running .dev version and passing local project and dataset directory as volume arguments, and then you run the setup manually inside the container.

**Comment by qyf13505049895** [comment]
Thank you for your answer. When I am processing the OSDAR23 dataset, it cannot be divided into a training set and a validation set. No matter how I set segment size, -- perm limit, and - p, the following error always appears: the loss generated by all threads is always 0
![image](https://github.com/egemenkopuz/temporal-bevfusion/assets/62499977/3101a42d-2487-4a97-9e1d-e72a6c3af27d)

**Comment by egemenkopuz** [comment]
The find_temporal_split algorithm utilizes exhaustive search, hence it may not always guarantee finding a split. Therefore, I recommend playing with constraint values and changing the seed. For example, consider reducing 'num-points-th' to 0.9 or even 0.85. Alternatively, you could manually split the data.

**Comment by Elegyt** [comment]
> 谢谢你的回答。我在处理OSDAR23数据集时，无法将其分为训练集和验证集。无论我如何设置segment size、--perm limit和-p，总是出现以下错误：所有线程产生的loss始终为0 ![image](https://github.com/egemenkopuz/temporal-bevfusion/assets/62499977/3101a42d-2487-4a97-9e1d-e72a6c3af27d)

Hello, is the problem solved? I have the same error as you.

![Image](https://github.com/user-attachments/assets/850d2634-df76-4427-90a8-6284965cb030)
